cluster-up/cluster/k8s-1.14.6/provider.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+image="k8s-1.14.6@sha256:ec29c07c94fce22f37a448cb85ca1fb9215d1854f52573316752d19a1c88bcb3"
```

Comment:
this sha is different from the one in cluster-provision/README.md?
(and yes, having these shasum spread through several files is ugly)

Reply:
Done